### ISSUE TITLE
unsupport plugin loading

### ISSUE THREAD

**Issue by z3t0** [voxel/voxel-drop#5]
>.js: loads JavaScript and/or plugins using voxel-plugins
.coffee: same as .js but first compiles CoffeeScript

Should we work on a better alternative to plugin loading? How does it deal with plugins that need to be browserified?

**Comment by deathcap** [comment]
Definitely, currently there is no browserification of plugins, which is a large problem. It's a complicated issue, largely unsolved, but some solution here would be necessary to support in-game plugin development. 

An attempt to browserify npm: https://github.com/deathcap/webnpm - some partial progress, downloads through a CORS proxy, but mainly nonfunctional so far, haven't looked at it in months.

https://github.com/jfhbrook/wzrd.in https://wzrd.in/ "browserify as a service" - this works, but calls out to a remote server for browserifying, not ideal. Maybe a good stop-gap solution until npm can run 100% in-browser, though.

**Comment by z3t0** [comment]
I see, so in the meantime should we disable loading of plugin via this plugin?

**Comment by deathcap** [comment]
I think it's still useful to leave as a placeholder until something better is developed, it'll work with very simple (no dependencies) plugins at least.

Another service to investigate, noticed this is now linked on npmjs.org "Test (packagename) in your browser": https://tonicdev.com/56a5548358155c0d0001d6f0/56bd901b9ccab40d00db69b4 - haven't looked into it, but maybe dynamic remote plugin loading could be implemented through it. Larger tracking issue in voxel-plugins: https://github.com/voxel/voxel-plugins/issues/1 Dynamic remote plugin loading?
